src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -72,6 +72,11 @@ configurationRegistry.registerConfiguration({
 			},
 			'default': []
 		},
+		'terminal.integrated.rightClickCopyPaste': {
+			'description': nls.localize('terminal.integrated.rightClickCopyPaste', "Controls whether copy/paste happens on mouse right click in integrated terminal."),
```

Comment:
I think the description needs to be more detailed, how about this?

> When set, this will prevent the context menu from appearing on right clicking in the terminal, instead it will copy when there is a selection and paste when there is no selection.